kustomize/overlays/azure/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: nodeless-cost-calculator
+  name: nodeless-cost-calculator
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+      - name: nodeless-cost-calculator
+        env:
+        - name: CLOUD_PROVIDER
+          value: azure
+        - name: REGION
+          value: "East US"
+      - name: cloudinfo
+        readinessProbe:
+          httpGet:
+            path: /api/v1/providers/azure/services/compute/regions/eastus/products
```

Comment:
this is crucial line. If scrape is now finished, cost-calculator will crash, so we mark cloudinfo container as ready once scraping is finished